examples/scenarioSunlineFiltersCompare.py

```diff
@@ -0,0 +1,352 @@
+
```

Comment:
:small_red_triangle_down: We need to add the copyright notice and license text here. Use the following for the copyright text.
"#  Copyright (c) 2025, Laboratory for Atmospheric and Space Physics, University of Colorado at Boulder"